support-frontend/assets/components/checkoutSupportOnce/checkoutSupportOnce.tsx

```diff
@@ -0,0 +1,98 @@
+import { css } from '@emotion/react';
+import {
+	brand,
+	from,
+	headline,
+	space,
+	textSans,
+	until,
+} from '@guardian/source-foundations';
+import { Box } from 'components/checkoutBox/checkoutBox';
+import type { ContributionType } from 'helpers/contributions';
+import { CheckoutSupportOnceButton } from './checkoutSupportOnceButton';
+
+const container = css`
+	border-radius: 12px;
+	margin: 10px ${space[3]}px ${space[4]}px;
+
+	${from.desktop} {
+		margin: ${space[5]}px ${space[6]}px;
+	}
+`;
+const top = css`
+	display: flex;
+	flex-direction: row;
+	margin-left: 0px;
+
+	${from.desktop} {
+		margin-left: 2px;
+	}
+`;
+const bottom = css`
+	display: flex;
+	flex-direction: row;
+	margin-left: 2px;
+
+	${from.desktop} {
+		margin-left: 0px;
+	}
+`;
+const heading = (color: string) => css`
+	${headline.xxsmall({ fontWeight: 'bold', lineHeight: 'regular' })};
+	color: ${color};
+
+	${from.desktop} {
+		${headline.xsmall({ fontWeight: 'bold', lineHeight: 'tight' })};
+	}
+`;
+const para = css`
+	${textSans.small({ lineHeight: 'regular' })};
+
+	${from.desktop} {
+		${textSans.medium({ lineHeight: 'regular' })};
+	}
+`;
+const box = css`
+	:not(:last-child) {
+		${until.tablet} {
+			margin-bottom: ${space[2]}px;
+		}
+	}
+	&:hover {
+		cursor: pointer;
+	}
+`;
+
+export type CheckoutSupportOnceProps = {
+	contributionType: ContributionType;
+	supportOnceDisplay: boolean;
+	supportOnceTitle: string;
+	supportOnceParagraph: string;
+	onSupportOnceClick: () => void;
+};
+
+export function CheckoutSupportOnce({
+	contributionType,
+	supportOnceDisplay,
+	supportOnceTitle,
+	supportOnceParagraph,
+	onSupportOnceClick,
+}: CheckoutSupportOnceProps): JSX.Element | null {
+	if (contributionType !== 'ONE_OFF' && supportOnceDisplay) {
+		return (
+			<Box onClick={onSupportOnceClick} cssOverrides={box}>
```

Comment:
@paul-daniel-dempsey I think we need to consider accessibility when making a `div` a clickable element. As it is it's essentially just a layout element (a `div`) with a click handler, I'm pretty this would be an issue for users using screen readers or keyboard navigation. This article covers this well: https://tech.boxed.com/accessible-ui-stop-making-clickable-divs-264115938716. They're suggestion is to use an HTML `button` element rather than a `div`.

Reply:
Hmmm, really struggling to get a button to follow the formatting of the original div. Buttons do not to appear to follow identical css rules as div's am finding. Is a label be an acceptable replacement as this does and its used in Choice Cards for example? Have put label version in and will continue attempting to figure out the button method. My CSS needs more practice!

Reply:
@paul-daniel-dempsey I'd be happy to pair on this if you'd like?

Reply:
My two cents on this:

- In the future we should resist this kind of pattern at the design stage. 'Clickable cards' are always a problem, and if we do not have a standardised UI pattern in Source for this kind of thing (or we are not creating the thing that designers want to make the standardised pattern) we should strenuously avoid doing it.
- Given that this element is sending users to another 'resource' (ie. the version of the page with single contributions available), the semantically correct element here is a link, not a button (buttons are for changing something or submitting information on the resource you're already on). There's a good article from CSS Tricks here on a few different ways to make a 'clickable card' with an internal `<a>` tag: https://css-tricks.com/creating-animated-clickable-cards-with-the-has-relational-pseudo-class/. However note that none of them are particularly great- hence why we should try and avoid this pattern if possible!